src/configuration/app-containers.md

```diff
@@ -362,10 +363,10 @@ languages:
 * Java (with integrated Maven and Ant support)
 
 Those dependencies are independent of the eventual dependencies of your
-application, and are available in the `PATH`, during the build process
+application and are available in the `PATH` during the build process
 and in the runtime environment of your application.
 
-You can specify those dependencies like this:
+You can specify those dependencies like below:
```

Comment:
The `below` keyword clearly indicates to the reader where they should look to find the dependencies.